README.md

```diff
@@ -74,8 +76,18 @@ takes advantage of features available only inside of the latest stable release.
 
 You need also need libvirt-dev(el) package installed.
 
+###Building The Provider
+
+Clone repository to: `$GOPATH/src/github.com/dmacvicar/terraform-provider-libvirt`
+
+```console
```

Comment:
I'm not sure of this doc.

The `go get` way is the standard way of golang, see for example

https://github.com/kubernetes/kubernetes#you-have-a-working-go-environment

Reply:
we can have a `git clone` approach but a Git clone approach would be needed in case we are **outside** gopath, since modules golang where implemented for that reason.

But imho if we decided to document the `module` approach with git clone, we should add it a second method  as the `gopath` one, which was historically what  golang offered first and still offer.

So imho I don't know if it is worth the effort because it complicated a little the doc providing 1, 2 approaches but if you feel to do it  ok. Imho we should just state to use `go get` as was before. What was your problem that led you to do so? In theory go get create automatically directory without need to create and git clone

Reply:
The problem comes when I want to build a specific version of this plugin. If you uses `go get` it always grabs the master. Which as of right now is build around using Terraform `0.12`, which my team is not using just yet. 

With `git clone`, you can pull down the repo. Then use a `git checkout` to pull down the correct tag version, aka `v0.5.2`. 

While you can do a similar thing with `go get` then a `git checkout` the go dependencies are not correct.

The only other way I know to do it is to use GO's new syntax, `go get <package>@<version>`. This requires `GO111MODULE=ON` and its not 100% stable in my experience.  

Hence, why I think the `git clone` is the best approach. Plus its what the official modules suggest, https://github.com/terraform-providers/terraform-provider-random#building-the-provider.

Reply:
I'm ok to add it